source/view.c

```diff
@@ -540,25 +553,95 @@ static void rofi_view_call_thread ( gpointer data, gpointer user_data )
     g_mutex_unlock ( t->mutex );
 }
 
+enum CharClass { LOWER, UPPER, DIGIT, NON_WORD };
+
+static enum CharClass char_class(gunichar c)
```

Comment:
can this be named more clearly:  something like: `rofi_sort_fzf_get_character_class`. Same for `scrore_for`.